node/pkg/ethereum/watcher.go

```diff
@@ -489,7 +494,7 @@ func (e *Watcher) Run(ctx context.Context) error {
 					zap.Stringer("current_blockhash", currentHash),
 					zap.Duration("took", time.Since(start)),
 					zap.String("eth_network", e.networkName))
-			}
+		}
```

Comment:
You removed the correct spacing and indented it strangely. Please just stick with `gofmt`

Reply:
Fixed